sktime/forecasting/var.py

```diff
@@ -0,0 +1,108 @@
+# -*- coding: utf-8 -*-
+__all__ = ["VectorAutoRegression"]
+__author__ = ["Taiwo Owoseni"]
+
+from statsmodels.tsa.api import VAR as _VAR
+from sktime.forecasting.base.adapters import _StatsModelsAdapter
+from sktime.utils.validation.series import check_series
+from sktime.utils.validation.forecasting import check_X
+import pandas as pd
+import numpy as np
+
+DEFAULT_ALPHA = 0.05
+
+
+class VectorAutoRegression(_StatsModelsAdapter):
```

Comment:
you should add docstrings. Complete lack of docstrings makes it difficult to review.